cls/SourceControl/Git/Settings.cls

```diff
@@ -75,6 +81,36 @@ Method %OnNew() As %Status
     quit $$$OK
 }
 
+Method SaveWithSourceControl() As %Status
+{
+    set sc = $$$OK
```

Comment:
Important! This method needs to new %SourceControl before calling SourceControlCreate.